apps/src/code-studio/components/DownloadReplayVideoButton.jsx

```diff
@@ -0,0 +1,172 @@
+import React, {PropTypes} from 'react';
+import i18n from '@cdo/locale';
+
+import color from "../../util/color";
+
+function downloadRemoteUrl(url, downloadName) {
+  fetch(url, {
+    method: 'GET',
+  })
+    .then(response => response.blob())
+    .then(blob => {
+      const element = document.createElement('a');
+      const url = URL.createObjectURL(blob);
+      element.setAttribute('href', url);
+      element.setAttribute('download', downloadName);
+      element.style.display = 'none';
+      document.body.appendChild(element);
+      element.click();
+      document.body.removeChild(element);
```

Comment:
This bit of code (creating a hidden element so that we can click on it to start a download I think?) feels like it could use a comment explaining itself.

Reply:
Sure thing! Is there a more standard way to initiate a user download from JS?

Reply:
Not that I know of, but I think this is enough of a "trick" that many won't be familiar with it/understand why this is being done when they see it in code.